_posts/2020-01-30-Age-Targeting-Part1.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: Increase your target audience via ML on Spark [part 1]
+subtitle: Ground truth selection
+category: data
+tags: [adtech, howto, spark]
+author: Alessio Borgheresi
+header-img: "images/Age-Targeting/true_age.jpg"
+---
+
+
+In mobile advertising it is possible to target the audience of the campaign based on some charateristics of the users.
+Here we are focusing on the age.
+However, these informations are only provided on by a subset of apps during the bidding process.
+To increase the available targeting audience we would need an estimation of the age for the devices which never use those apps and here is when the machine learning comes into place.
+
+To train a supervised machine learning for infearing the age over all the devices we need: 
+- a **reliable ground truth** datasets (in our case the age for certain devices);
+- the **features** which will go as inputs to the model.
+
+The ground truth has to be reliable or the model will learn non realistic structure.
+Moreover, it could improve the performances of the training.
```

Comment:
negate sentence -> an unreliable ground truth, which is based on arbitrary or random relations will decrease the performance of the model